app/modules/auth/utils/validate.js

```diff
@@ -0,0 +1,56 @@
+export function isEmpty(str) {
+    return (!str || 0 === str.length);
+}
+
+export function validateEmail(email) {
+    var filter = /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+
+    if (filter.test(email)) return true;
+
+    return false;
+}
+
+export function validatePassword(password) {
+    if (password.length > 6) return true;
```

Comment:
could you just `return password.length > 6`? or is there a reason behind you doing this?

Reply:
Can be applied to other parts as well.

Reply:
nope I should change